docs/sphinx/internals/glossary.rst

```diff
@@ -83,10 +83,10 @@ RTP
     Synchronization source. Identifies a stream inside a session and should be unique only inside that session.
 
 *seqnum*
-    Sequence number. Identifies a packet inside a stream. All packets in a stream are numbered sequentially usually starting from a random number.
+    Sequence number. Identifies a packet inside a stream. All packets in a stream are numbered sequentially usually starting from a cryptographically strong random number.
```

Comment:
I think we should omit "cryptographically strong" here. First, it's not necessary for understanding the terms (seqnum and timestamp). Second, it's not quite true: we use CSPRNG when possible (libuv > 1.33), but not necessary. It's an important detail. We should document it separately. I think in future we should add a page with security concerns. Third, "cryptographically strong" is a property of random *generator*, not a random *number*.